packages/cli/src/commands/add.ts

```diff
@@ -81,7 +81,7 @@ export default class AddCommand extends Command {
 
     if (isLocalHost) this.warn('`localhost` network detected, prompting user for inputs');
 
-    let startBlock = startBlockFlag;
+    let startBlock = parseInt(startBlockFlag ?? '').toString();
```

Comment:
if `startBlockFlag` is not provided this results in NaN and the whole command fails